libs/core/async_cuda/include/hpx/async_cuda/cuda_event.hpp

```diff
@@ -86,7 +68,33 @@ namespace hpx { namespace cuda { namespace experimental {
             return free_lists_[device].push(event);
         }
 
+        // delete copy / move constructors
+        cuda_event_pool(cuda_event_pool&&) = delete;
+        cuda_event_pool& operator=(cuda_event_pool&&) = delete;
+        cuda_event_pool(const cuda_event_pool&) = delete;
```

Comment:
just a nitpick - we're using 'east const' in all the code base. Would you mind adapting this?

Reply:
Ah, I forgot about that! Done!